retis/src/core/inspect/btf.rs

```diff
@@ -92,10 +92,7 @@ impl BtfInfo {
     pub(crate) fn resolve_types_by_name(&self, name: &str) -> Result<Vec<(&Btf, Type)>> {
         let mut types = Vec::new();
 
-        let mut base_types = match self.vmlinux.resolve_types_by_name(name) {
-            Ok(base_types) => base_types,
-            _ => Vec::new(), // Id not found in base.
-        };
+        let mut base_types = self.vmlinux.resolve_types_by_name(name).unwrap_or_default();
```

Comment:
@liuhangbin,

this could be dropped.

See:
https://github.com/retis-org/retis/pull/425/commits/123fcf9191ab40c6b736c5641ba95053df39edc8

Reply:
Ah, I forgot to rebase. Thanks for this info